### ISSUE TITLE
[BUG] Can't switch to a new show.

### ISSUE THREAD

**Issue by Sodie1** [nixgates/plugin.video.seren#859]
**Describe the bug**
Seren 3.0 on an updated firestick 4k.  Kodi is also up to date. Since the latest Seren update if I exit a show I can no longer go into recently watched without rebuilding the Trakt database. I click recently watched and the loading wheel comes up, then the screen flashes and it stays on the same page. 

**To Reproduce**
1. Go to a show and exit it. 
2. Attempt to go pick a new show. 

Thanks for all of your hard work, you all rock.

**Comment by drinfernoo** [comment]
Please attach a debug log.

**Comment by nixgates** [comment]
I believe this may be related to #864. 

Does it work on menus outside of My Shows by chance?

**Comment by Sodie1** [comment]
> I believe this may be related to #864.
> 
> Does it work on menus outside of My Shows by chance?

Yes sir. It's definitely update specific. I didn't update Seren on my tablet and it's fine. I updated on my firestick with a skin applied and have issues. Tried on my phone without a skin but updated seren and I have issues. So I ruled out it being the skin.

**Comment by Sodie1** [comment]
> I believe this may be related to #864.
> 
> Does it work on menus outside of My Shows by chance?

Just read up in that issue you linked. This happens with shows not marked as watched on any other device or service.

**Comment by nixgates** [comment]
@Sodie1 yea, the original thought on that other issue was a little off and it will occur on menus regardless of a skin or marking on another device.

When I recreated the issue in the linked post I did recreated in a manner that was nearly exactly like you described.

We have a fix in testing at the moment, we'd like to test it for about a week as the sync is very central to the addons functionality. Once we are 100% sure we didn't break anything we will push it out :)

**Comment by runkpocker** [comment]
Reading through related bugs and it sounds like this one has been sorted out. My experience is the exact same, problem when on My Shows on a firetv, causes traktsync.db locking.
